decayAmplitude/decayTopology.cc

```diff
@@ -145,9 +145,10 @@ decayTopology::doClone(const bool cloneFsParticles,
 		// clone vertex
 		if (   isDecayVertex(vert)
 		    or isProductionVertex(vert)
-		    or (cloneFsParticles and isFsVertex(vert)))
+		    or (cloneFsParticles and isFsVertex(vert))) {
 			topoClone->isDecayVertex(vert);
-			newVerts.push_back(topoClone->cloneNode(*iNd));
+		}
+		newVerts.push_back(topoClone->cloneNode(*iNd));
```

Comment:
Line 149 does not have any effect, so with your current proposal the complete ```if``` statement including its action (lines 146 to 150 with the patch) could be removed. Are you sure this is correct? Probably the indentation checker really found a bug here.